Cargo.toml

```diff
@@ -53,20 +53,22 @@ alloy = { version = "=1.0.23", default-features = false, features = [
   "contract",
   "k256",
   "std",
-] } # pinned to 1.0.4 due to many breaking changes in patch releases
+] } # pinned to 1.0.16 due to many breaking changes in patch releases
 anyhow = "1.0.98"
 aquamarine = "0.6.0"
 arrayvec = { version = "0.7.6", features = ["serde"] }
+async-compression = "0.4.27"
 async-channel = "2.5.0"
 async-lock = "3.4.0"
 async-signal = "0.2.12"
 async-stream = "0.3.6"
+async-tar = "0.5.0"
 async-trait = "0.1.88"
 asynchronous-codec = { version = "0.7.0", features = ["cbor"] }
 atomic_enum = "0.3.0"
 axum = { version = "0.8.4", features = ["ws", "http2"] }
 axum-extra = { version = "0.10.1", features = ["query"] }
-backon = { version = "1.5.1", default-features = false }
+backon = { git = "https://github.com/tolbrino/backon", branch = "tb/202507-expose-future-timer-sleep", default-features = false }
```

Comment:
Please annotate with a comment why this has to be a modified fork.

Also, please pin to the specific commit hash, not a branch that can move the tip of the HEAD>

Reply:
Moved to the latest upstream version, which includes the bug-fix.